src/classes/audioContextWrapper.tsx

```diff
@@ -47,6 +52,7 @@ class AudioContextWrapper {
 
     this.masterGainNode.connect(this.audioContext.destination);
     this.waveform = getWave(waveformId);
+    this.wavetable = WavetableCache.getSingleton().get("organ") ?? Wavetable.createEmpty();
```

Comment:
This is the current state setup - setting all 'custom' waves to organ, right? With future intention of opening this up?

Reply:
Also I'm good shipping this as is, but I think we'd benefit from a waveTable service layer that takes declarative calls and returns the requested wavetable.